air-script/tests/random_values/random_values_simple.air

```diff
@@ -0,0 +1,18 @@
+def RandomValuesAir
+
+trace_columns:
+    main: [a, b]
+    aux: [c, d]
+
+public_inputs:
+    stack_inputs: [16]
+
+random_values:
+    rand: [16]
+
+integrity_constraints:
+    enf c' = $rand[15] - $rand[0] + $rand[3]
+
+boundary_constraints:
+    enf c.first = $rand[5] + $rand[3] + $rand[15]
+    enf c.last = $rand[0] + $rand[15] + $rand[11]
```

Comment:
nit: let's swap the order of the `boundary_constraints` and `integrity_constraints`